generators/entity-server/files.js

```diff
@@ -196,6 +191,20 @@ const serverFiles = {
                 },
             ],
         },
+        {
+            condition: generator => generator.reactive && ['sql'].includes(generator.databaseType) && !generator.embedded,
+            path: SERVER_MAIN_SRC_DIR,
+            templates: [
+                {
+                    file: 'package/repository/EntityReactiveRepositoryInternalImpl.java',
+                    renameTo: generator => `${generator.packageFolder}/repository/${generator.entityClass}RepositoryInternalImpl.java`,
```

Comment:
The purpose of an internal repository is to be internal and thus hidden. So this class should not be public and should be in `${generator.entityClass}Repository.java` IMO.
Note that it could still be in a separate template, but it should be included in the other template so there's only one java class at end.

Reply:
This seems to be the only unresolved comment on this PR. Can you please have a look @gzsombor?